product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -352,7 +352,7 @@ def get_products_to_print(self):
 
     def get_group_key(self, product):
         group_field = getattr(product, self.group_field)
-        complete_name = getattr(group_field, "complete_name", group_field.name) or _(
+        complete_name = getattr(group_field, "translatable_complete_name", False) or getattr(group_field, "complete_name", group_field.name) or _(
```

Comment:
```suggestion
        complete_name = getattr(group_field, "translatable_complete_name", group_field.name) or _(
```

Reply:
it's not only get name from product.category. we can choose `Group Field`, so we still use attribute `complete_name` and `name`
![image](https://github.com/chaule97/product-attribute/assets/32189890/32c22581-ec9e-4fc8-92cb-799ea33fd387)